Makefile

```diff
@@ -157,7 +157,7 @@ upload-openstack-image:
 	ATLAS_PROJECT=${ATLAS_PROJECT} S3_TARGET=$(S3_TARGET) ./scripts/openstack-s3-upload.sh
 
 docker-build:
-	docker build -t images:build - < Dockerfile.build
+	docker build -t cloudbreak-centos7 -f docker/centos7.3/Dockerfile .
```

Comment:
NAT tests are running on CentOS 7.4 as I know.

Reply:
CentOS-7.4 was introduce 1.5 weeks ago. I'll make CentOS-7.4 support in a separate round.